cmd/find-main.go

```diff
@@ -0,0 +1,141 @@
+/*
+ * Minio Client (C) 2014, 2015 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+)
+
+// find specific flags
+var (
+	findFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "Find files of a matching name",
+		},
+		cli.StringFlag{
+			Name:  "path",
+			Usage: "Find global matches on the entire path",
+		},
+		cli.StringFlag{
+			Name:  "regex",
+			Usage: "Matches all files with PCRE regular expression",
+		},
+		cli.StringFlag{
+			Name:  "print",
+			Usage: "Prints the found files given args to STDOUT",
+		},
+		cli.StringFlag{
+			Name:  "mcexec",
```

Comment:
The rationale for calling it --mcexec is that the command has to be an mc command (vs any shell command which would be implied by --exec).

Why? 

mc find myminio -name *.jpg

will return something like

myminio/somebucket/somefolder/x1.jpg
mymino/somebucket/somefolder/x2.jpg

then names returned (if the are names of objects in an object store) are not understandable to a standard unix/linux shell command, so 

mc find myminio --name *.jpg --exec "cp {} ~/pictures" 

Fails because the shell cp command does not work on objects in myminio... 

--mcexec makes it more obvious/clear that the command has to be an mc command.